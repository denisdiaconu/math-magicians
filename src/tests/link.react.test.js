import React from 'react';
import renderer from 'react-test-renderer';

it('Render works correctly', () => {
  const tree = renderer.create(<div>buttons</div>).toJSON();
  expect(tree).toMatchSnapshot();
});
