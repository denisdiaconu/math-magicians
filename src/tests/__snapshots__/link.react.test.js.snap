// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render works correctly 1`] = `
<div>
  buttons
</div>
`;
